benchmark/common.js

```diff
@@ -1,5 +1,8 @@
-var assert = require('assert');
+var fs = require('fs');
 var path = require('path');
+var assert = require('assert');
+var spawn = require('child_process').spawn;
```

Comment:
Minor style nit: I don't think it's a hard rule but imports tend to be sorted in alphabetical order.